files/contaodemo/theme/src/scss/components/navigation/_nav-main.scss

```
@@ -246,7 +246,7 @@ $s-underline-size: .125rem;
         background: #fff;
         box-shadow: 1px 2px 15px rgba(0, 0, 0, .1);
         gap: 1rem;
-        z-index: 1;
+        z-index: 2;
```

```suggestion
        z-index: 2; // Higher z-index due to swiper
```

Would add a comment on that for clarity 😊